hack/common.sh

```diff
@@ -217,6 +229,16 @@ os::build::build_binaries() {
           "$(dirname ${test})"
       done
 
+      for artifact in ${binaries[@]}; do
```

Comment:
i tried getting rid of this, but the explicit build step still seems to be required? unless i missed something.